### DISCUSSION TITLE
i have a question！

### DISCUSSION THREAD

**Discussion by princemero** [toeverything/AFFiNE#1519] [General]
Now the private deployment does not yet support collaborative software, I configure it to open in local port browser without setting and collaboration button, if I have LAN environment, how can I collaborate with them together? By the way, after local deployment, do I still need to login to Google account for collaborative work?

**Comment by darkskygit** [comment]
If you mean `ghcr.io/toeverything/affine-self-hosted:alpha-abbey-wood`, this version does not yet support collaboration. We have been supporting collaboration since the `Downhills` version. At present, we have not officially released the self-hosted version of `Downhills`, but you can compile the code or go to `https://app.affine.pro` to experience it. We will release the self-hosted version of the `Downhills` version in the near future, but there are still some development work that has not been completed. Please stay tuned.

**Comment by Yipei-Operation** [comment]
> Now the private deployment does not yet support collaborative software, I configure it to open in local port browser without setting and collaboration button, if I have LAN environment, how can I collaborate with them together? By the way, after local deployment, do I still need to login to Google account for collaborative work?

As for our current docker user guide, can refer to this link: https://community.affine.pro/c/user-guide/self-host-affine-with-docker-pre-alpha-and-alpha-local-setup-user-guide
we are working hard on the docker image of the downhills version, and the client of the desktop, we hope can launch them within 1 month:
Any questions welcome to contact us via our social media:
* Reddit: https://www.reddit.com/r/Affine/
* Discord: https://discord.gg/Arn7TqJBvG
* Telegram: https://t.me/affineworkos
* Twitter: https://twitter.com/AffineOfficial
* Medium：https://medium.com/@affineworkos
* Community：https://community.affine.pro/home
* Github: https://github.com/toeverything/AFFiNE
